### DISCUSSION TITLE
Generic question about binding

### DISCUSSION THREAD

**Discussion by phretor** [Koenkk/zigbee2mqtt#9738] [General]
Is binding something that can be added support for, or has to be built in into a device's native firmware?

What I don't understand is why some wall switches like the ZM-L03E-Z can bind to useless clusters like "Ota" and not "OnOff".

If this is something Z2M can't intervene on, is there a way to search the DB of supported devices and filter for those that support binding?

Thanks and keep up with the great work!

**Comment by Koenkk** [comment]
Binding is a device firmware feature, z2m doesn't have any influence on that.


> is there a way to search the DB of supported devices and filter for those that support binding?

Whether a device supports binding is sometimes listed on the device page itself but ofcourse depends on the community to add this information.
